tests/server/routers/test_structures.py

```diff
@@ -67,6 +67,15 @@ def test_structures_endpoint_data(self):
         assert "_exmpl_chemsys" in self.json_response["data"]["attributes"]
 
 
+def TestCheckResponseSingleStructure(check_response):
```

Comment:
```suggestion
def test_check_response_single_structure(check_response):
```

The other tests cases are named in camel case as they are classes, if you have a test function it should be snake case like this.